src/store/index.js

```diff
@@ -165,25 +159,28 @@ export default createStore({
     addToCart(state, payload) {
       this.commit("ADD_TO_CARD", payload);
     },
-    delFromCart(state) {
-      this.commit("DEL_FROM_CARD");
+    delFromCart(state, payload) {
+      this.commit("DEL_FROM_CARD", payload);
     },
   },
   getters: {
     totalPrice(state) {
-      return state.shoppingCart[0].products[0].points;
+      return state.shoppingCart[0].totalPrice;
     },
     getUserPoints(state) {
       return state.shoppingCart[0].userPoints;
     },
     pointsAfterShopping(state) {
       return (
-        state.shoppingCart[0].userPoints -
-        state.shoppingCart[0].products[0].points
+        state.shoppingCart[0].userPoints - state.shoppingCart[0].totalPrice
       );
     },
     productInCart(state) {
-      return state.shoppingCart[0].products[0].name;
+      const prods = [];
+      for (let i = 0; i < state.shoppingCart[0].products.length; i++) {
+        prods.push(state.shoppingCart[0].products[i].name);
+      }
+      return prods.join(", ").slice(0, -2);
```

Comment:
A nie możesz zrobić po prostu gettera, który weźmie przedmioty z koszyka i zliczy jak duża jest tablica?
Nie rozumiem po prostu dlaczego tu jest stała prods i dodajesz produkty w koszyku - co jest świetne, ale 
spróbuj zrobić po prostu gettera, który zliczy ile jest produktów